Meadowlark-js/packages/meadowlark-core/src/message/DeleteRequest.ts

```diff
@@ -10,7 +10,7 @@ import { Security } from '../security/Security';
 export type DeleteRequest = {
   documentUuid: DocumentUuid;
   resourceInfo: ResourceInfo;
-  validate: boolean;
+  validateNoReferencesToDocument: boolean;
   security: Security;
```

Comment:
I realized while working on update that our `validate` term covers two different kinds of validations. This one is that nothing references the document to be deleted.